javascript/javascript1/week3/homework3.js

```diff
@@ -109,18 +109,21 @@ saveNote("Make dinner", 3);
 console.log(notes); // [{content: 'Pick up groceries', id: 1}, {content: 'Do laundry', id: 2}]
 
 // Get a note
+let neddedId = -1;
 
 function getNote(id) {
     for (i = 0; i < notes.length; i++) {
-        if (notes.i === id) {
+        if (notes[i].id === id) {
+            neddedId = notes[i].id;
             return notes[i];
-        } else {
-            console.log("You dont have this note");
         }
     }
+    if (neddedId === -1) {
```

Comment:
@vulk29 Thank you for review =)  can you look on my changes please? I hope i get what you meant

Reply:
yup! this is exactly what i had in mind! Excellent job!